src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -162,6 +162,8 @@ public function normalize($object, $format = null, array $context = [])
                     }
                 }
 
+                $pathOperation = $this->updateGetOperation($pathOperation, $mimeTypes, $subresourceOperation['collection'] ? OperationType::COLLECTION : OperationType::ITEM, $subResourceMetadata, $subresourceOperation['resource_class'], $subresourceOperation['shortNames'][0], $operationName, $definitions);
```

Comment:
I'm really unsure. A subresource is a new operation, why should it inherit from another operation?

Reply:
In functionality it already does (i.e. the pager options do work without this PR, you can just put page=X in the URL and it will give you page X), only in documenting itself it does not. These are attributes that are not specific to an operation, but to the entire entity, which are (partially) ignored now.

I don't know if inherit is the right term, or if this is really the best solution to the issue. I'm open to alternative approaches.